Framework/DataHandling/src/DownloadInstrument.cpp

```diff
@@ -87,13 +87,13 @@ void DownloadInstrument::exec() {
   } catch (Mantid::Kernel::Exception::InternetError &ex) {
     std::string errorText(ex.what());
     if (errorText.find("rate limit") != std::string::npos) {
-      g_log.notice() << "Instrument Definition Update: " << errorText << '\n';
+      g_log.notice() << "Instrument Definition Update: " << errorText << "\n";
```

Comment:
Why are you changing `'\n'` to "\n"? The former is half the size and slightly more precise.